homeassistant/components/light/tcpbulbs.py

```diff
@@ -0,0 +1,146 @@
+"""
+Support for TCP Connected (Greenwave Reality) lights.
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.tcpbulbs/
+"""
+import logging
+import voluptuous as vol
+# Import the device class from the component that you want to support
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, Light, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS)
+from homeassistant.const import CONF_HOST
+import homeassistant.helpers.config_validation as cv
+# Home Assistant depends on 3rd party packages for API specific code.

```

Comment:
Unnecessary comment, applies also to comments above and below.

Reply:
Those are left over from the template, will remove.